packages/yew-platform/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "yew-platform"
```

Comment:
What if we dropped the `yew-` prefix from here? It can be generic crate that can be used to pick an async runtime between tokio and wasm-bindgen-futures (JS promises)

Name idea: promise + tokio = prokio

Reply:
Sounds good. I have published the crate under name `prokio`.